core/com.b2international.snowowl.core.rest/src/com/b2international/snowowl/core/rest/ControllerExceptionMapper.java

```diff
@@ -50,11 +52,21 @@
 		if (Throwables.getRootCause(ex).getMessage().toLowerCase().contains("broken pipe")) {
 	        return null; // socket is closed, cannot return any response    
 	    } else {
-	    	LOG.trace("Exception during request processing", ex);
+	    	if (PlatformUtil.isDevVersion()) {
```

Comment:
If anyone is wondering (I sure was): https://stackoverflow.com/questions/2621701/setting-log-level-of-message-at-runtime-in-slf4j/7829534

Quick summary: there is no way to do this programmatically in SLF4j without a helper method.